### ISSUE TITLE
Spell Level (D&D 5e)

### ISSUE THREAD

**Issue by liquidox** [Roll20/roll20-api-scripts#421]
So what does this script do?

https://wiki.roll20.net/Script:Spell_Level_(D%26D_5e) is empty.

**Comment by bpunya** [comment]
Here's a link to the script itself. It seems to automatically update spell cantrip damage as players level up.
https://github.com/Roll20/roll20-api-scripts/tree/master/SpellLevel5e

**Comment by shdwjk** [comment]
Yeah, it was a script to determine the cantrip damage programmatically.  It's basically been superseded by a math formula that does the same thing without the API.  I should probably remove it...
